dev/com.ibm.ws.ejbcontainer.bindings_fat/publish/servers/com.ibm.ws.ejbcontainer.bindings.fat.serverxml.testserver/server.xml

```diff
@@ -0,0 +1,53 @@
+<server>
+    <featureManager>
+        <feature>servlet-3.1</feature>
+        <feature>ejbRemote-3.2</feature>
+        <feature>ejbHome-3.2</feature>
+		<feature>componenttest-1.0</feature>
+    </featureManager>
+    
+    <include location="../fatTestPorts.xml"/>
+
+    <iiopEndpoint id="defaultIiopEndpoint" iiopPort="${bvt.prop.IIOP}" iiopsPort="${bvt.prop.IIOP.secure}"/>
+    
+    <ejbContainer bindToServerRoot="true"/>
+    
+    <application location="ServerXMLTestApp.ear">
+    
+    	<ejb-jar-bnd moduleName="BindingNameEJB">
+    		<session name="BindingName5">
+    			<interface class="com.ibm.ejb3x.BindingName.ejb.BindingNameIntf" binding-name="ejblocal:ejb/ServerXMLBindingNameIntf5"/>
+  				<interface class="com.ibm.ejb3x.BindingName.ejb.RemoteBindingNameIntf" binding-name="ejb/ServerXMLBindingNameIntf5"/>
```

Comment:
I hate to be that guy, but could this file be reformatted with a uniform indentation scheme?  I normally wouldn't necessarily protest about it, but since this test is actually about the server.xml file we should make sure it's easy to read.

Reply:
Being that guy is ok, fixed it.